radmon/verf_radtime.fd/bad_penalty.f90

```diff
@@ -0,0 +1,100 @@
+!$$$ subprogram documentation block
+!               .      .    .
+! subprogram:	penalty			build penalty.txt file
+!     prgmmr:   safford			date:  2009-11
+!
+! abstract:	This module contains code to build the penalty.txt file.
+!		The penalty.txt file reports the satellite and channel for
+!		which a penalty value is likely out of range.
+!
+! program history log:
+!	2009-11-23 safford - initial coding
+!
+! contains:
+!
+!
+! attributes:
+!   language:  f90
+!   machine:   ibm RS/6000 SP
+!
+!$$$
+
+module bad_penalty
+
+  implicit none
+  
+  private
+
+  public :: open_bad_penalty_file
+  public :: write_bad_penalty
+  public :: close_bad_penalty_file
+
+  integer, parameter       :: funit = 11
+
+  contains
+
+
+    !-------------------------------------------------------------
+    !  open the bad_pen file
+    !-------------------------------------------------------------
+
+    subroutine open_bad_penalty_file( date, cycle, fios )
+
+      !--- interface
+      character(8), intent( in )	:: date
+      character(8), intent( in )	:: cycle
+      integer, intent( out )		:: fios     
+     
+      !--- variables
+      logical                           :: lexist = .FALSE.
+      character(60)                     :: fname
+      
+!      write(*,*) '--> open_bad_penalty_file, date, cycle = ', date, cycle
```

Comment:
This looks like commented code leftover from debugging.  Perhaps it should be removed?